lib/generators/decidim/templates/app.json.erb

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "<%= app_name %>",
+  "description": "TODO: Add a short description about <%= app_name %>",
+  "keywords": [
+    "small",
+    "sharp",
+    "tool"
+  ],
+  "addons": [],
```

Comment:
Maybe we should add default addons, such as Redis, Memcache, Postgres, Rollbar, New Relic, Sendgrid